values/gitea/gitea-otomi-db.gotmpl

```diff
@@ -9,6 +9,7 @@ name: gitea-db
 storage:
   size: {{ $gdb.size }}
 instances: {{ $gdb.replicas }}
+imageName: ghcr.io/cloudnative-pg/postgresql:15.10
```

Comment:
Do we also want gitea and keycloak to be default at 17.0? And only leave the clusters that currently are on 15 stay on 15?
Then we don't have to upgrade that much clusters manually...